### DISCUSSION TITLE
Lipo battery charge

### DISCUSSION THREAD

**Discussion by teddybee** [strid3r21/BeeS3#3] [Q&A] [answered]
I am curious, can this board charge the lipo battery via usb? Does it have overcharge protection?

Can I connect a little solar panel as well to it?

**Comment by strid3r21** [comment, marked as answer]
> I am curious, can this board charge the lipo battery via usb? Does it have overcharge protection?
> 
> Can I connect a little solar panel as well to it?

No, the bees3 does not have lipo charging circuitry built in.
